cluster/gce/config-default.sh

```diff
@@ -201,3 +201,6 @@ ENABLE_DEFAULT_STORAGE_CLASS="${ENABLE_DEFAULT_STORAGE_CLASS:-true}"
 # TODO(dawn1107): Remove this once the flag is built into CVM image.
 # Kernel panic upon soft lockup issue
 SOFTLOCKUP_PANIC="${SOFTLOCKUP_PANIC:-false}" # true, false
+
+# Indicates if the values (eg. kube password) should be treated as canonical
```

Comment:
add: " in metadata, and therefore whatever is on disk ought to be recreated / clobbered."